spec/figure_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../lib/figures/figure'
+require_relative '../lib/figures/king'
+
+describe Figure do
```

Comment:
You claim the test is about `Figure`, but then test `King`. That's just a lie xD
Please test the class you claim to be testing.